PyInstaller/depend/bindepend.py

```diff
@@ -41,6 +45,9 @@
     import pefile
     # Do not load all the directories information from the PE file
     pefile.fast_load = True
+    is_win_10 = (platform.win32_ver()[0] == '10')
+else:
+    is_win_10 = False
```

Comment:
This is unused and should be removed (mind removing the import above, too). If it is to be kept for some reason, it  should go into PyInstaller.compat like this
```
is_win_10 = is_win and platform.win32_ver()[0] == '10'
```

Reply:
Removed. Belongs to another PR.